packages/picasso.js/src/core/chart/index.js

```diff
@@ -233,6 +234,22 @@ import componentCollectionFn from './component-collection';
  * @property {number} verticalMoving
  */
 
+/**
+ * A function which returns either (1) false (to specify no progressive rendering used) or an object specifing the data chunk rendered.
+ * @callback {function|undefined} RendererSettings~Progressive
+ * @returns {ProgressiveObject|boolean}
+ * @experimental
+ */
+
+/**
+ * A format to represent a data chunk to be rendered.
+ * @typedef {object} ProgressiveObject
```

Comment:
Since the consuming setting for this definition is experimental. I think this should also be `@experimental`. As I think any change to this object later will be threated as a breaking change which my not be what we want.

Reply:
Updated!